pkgs/development/python-modules/channels_redis/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, redis, msgpack, channels, aioredis, async_generator, pytest
+}:
+buildPythonPackage rec {
+
+  pname = "channels_redis";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "django";
+    repo = pname;
+    rev = version;
+    sha256 = "05niaqjv790mnrvca26kbnvb50fgnk2zh0k4np60cn6ilp4nl0kc";
+  };
+
+
+  propagatedBuildInputs = [
+    redis
+    aioredis
+    msgpack
+    channels
+  ];
+
+  checkInputs = [ pytest async_generator ];
```

Comment:
I'm seeing more testing requirements here https://github.com/django/channels_redis/blob/2.4.0/setup.py#L12 (specifically cryptography).

Are then not needed in the code anymore?

Reply:
The path that uses `cryptography` (https://github.com/django/channels_redis/blob/6f6ced8afa126b533fef32461b2ecfd1f8f5a0a3/channels_redis/core.py#L256) doesn't appear to be covered by the tests.

Reply:
yep it is an extra feature
* https://github.com/django/channels_redis/blob/6f6ced8afa126b533fef32461b2ecfd1f8f5a0a3/setup.py#L39

I'd leave that up to the author to decide whether they want it enabled default or not.